Tests/Marketplace/search_and_install_packs.py

```diff
@@ -269,17 +280,22 @@ def search_pack_and_its_dependencies(client, prints_manager, pack_id, packs_to_i
         lock (Lock): A lock object.
     """
     pack_data = []
-
     if pack_id not in packs_to_install:
         pack_display_name = get_pack_display_name(pack_id)
         if pack_display_name:
             pack_data = search_pack(client, prints_manager, pack_display_name, thread_index, lock)
+        if pack_data is None:
```

Comment:
when will it happen?

Reply:
What I found is that if it's a new pack, or premium pack, and doesn't currently exist in the marketplace bucket, it won't find an `id` or `version`. This was a way of keeping the existing install logic while remedying that.